css/camera_controls.css

```diff
@@ -0,0 +1,20 @@
+.cc-container {
+    display: flex;
+    flex-direction: column;
+    position: absolute;
+    right: 40px;
+    bottom: 40px;
+    gap: 3px;
+}
+
+.cc-container .btn {
+    border: none;
+    color: #d8d8d8;
```

Comment:
maybe another PR, but it's nice to have colors in a single file and use them as variables so they're reusable across components

Reply:
at first pass this looks like it will take a bit of configuring so i'll make a separate issue for it